staging/src/k8s.io/cli-runtime/pkg/printers/omit.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
I'd probably name this file `managedfields.go`.

Reply:
@soltysh I have updated the file name.